.github/workflows/onPushToMain.yml

```diff
@@ -0,0 +1,48 @@
+name: Push to main
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  tests:
+      uses: salesforcecli/github-workflows/.github/workflows/unitTest.yml@main
+  release:
+    runs-on: ubuntu-latest
+    container:
+      image: node:lts
+    needs:
+    - tests
+    steps:
+    - uses: actions/download-artifact@v2
+    - uses: actions/checkout@v3
+    - name: Configuring GitHub
+      run: |-
+        git config credential.helper 'cache --timeout=120'
+        git config user.email "$GH_EMAIL"
+        git config user.name "Release Bot"
+    - name: Bump package version
+      run: |-
+        yarn
+        yarn bump-versions 'minor'
+        git add .
+        export RELEASE_TAG="$(node -pe "require('./lerna.json').version")"
+        git commit -m "Updated version $RELEASE_TAG [ci skip]"
+    - name: Set .npmrc
```

Comment:
Since this is in onRelease, Set .npmrc and the yarn publish-lsp and git push origin main steps can all be removed here